cargo-near/src/commands/build_command/docker/docker_checks/hello_world.rs

```diff
@@ -0,0 +1,40 @@
+use colored::Colorize;
+
+const ERR_SANITY: &str = "`docker` sanity check failed!";
+
+const PERM_DENIED_STATUS: i32 = 126;
+
+pub fn check() -> color_eyre::eyre::Result<()> {
+    let mut docker_cmd = std::process::Command::new("docker");
+    docker_cmd.args(["run", "hello-world"]);
+    let output_result = docker_cmd.output();
+    let output = super::handle_command_io_error(
+        &docker_cmd,
+        output_result,
+        color_eyre::eyre::eyre!(ERR_SANITY),
+    )?;
+
+    if !output.status.success() {
+        let stderr = std::str::from_utf8(&output.stderr)?;
+        println!();
+        println!("{}", stderr.yellow());
+        if permission_denied(&output.status, stderr)? {
+            println!("{}", "Permission denied!".cyan());
+            super::print_installation_links(true);
```

Comment:
nit: This `bool` argument drives me crazy. Let's just extract that printing into a separate function and call it separately here. It is the only place where `true` is passed.